frontend/www/js/omegaup/login/logout.ts

```diff
@@ -0,0 +1,48 @@
+import { OmegaUp } from '../omegaup';
+
+OmegaUp.on('ready', () => {
+  const clientId = document.querySelector(
+    'meta[name="google-signin-client_id"]',
+  );
+  if (!clientId) {
+    redirect();
+    return;
+  }
+
+  interface Reason {
+    error: string;
+    details: string;
+  }
+
+  // All possible paths need to end with redirect().
+  gapi.load('auth2', function() {
+    // ['then'] is used instead of .then(), since these are not real ES6
+    // Promise objects, therefore they don't have an .else() or .finally().
+    // That trips up the linter.
+
+    gapi.auth2.init({})['then'](
+      (auth: gapi.auth2.GoogleAuth) => {
+        auth.signOut()['then'](
+          () => {
+            redirect();
+          },
```

Comment:
nit:

```suggestion
          () => redirect(),
```

cuando es sólo una expresión, se prefiere usar la forma corta de las arrow functions. igual con todas las demás arrow functions de este archivo.

Reply:
listo